### PR TITLE
LRQA-68823

### PR DESCRIPTION
**Pull request by gleicelisbino**

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 1]
```diff
@@ -501,22 +501,9 @@ definition {
 
 	macro editTextBodyToolBar {
 		SelectFrame.selectFrameNoLoading(locator1 = "CKEditor#BODY_FIELD_IFRAME");
-
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:504]: @gleicelisbino, let's keep this blank line to separate the instructions. :wink:

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro [hunk 2]
```diff
@@ -978,6 +965,27 @@ definition {
 			locator1 = "Form#SIDEBAR_ADVANCED_PREDEFINED_VALUE");
 	}
 
+	macro toggleCKEditorContentStyle {
+		if (isSet(content)) {
+			FormFields.editTextBodyToolBar(content = "${content}");
+		}
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:971]: @gleicelisbino, can we keep the macro `toggleCKEditorContentStyle` only in charge of toggle the content styles?

In the test, we invoke the macro `editTextBodyToolBar` to insert the content `Test`, then we invoke the macro `toggleCKEditorContentStyle` to enable and disable the content styles every time it's needed. :wink:

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro (hunk 1 above)
Comment (gleicelisbino) [reply portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:504]: @natocesarrego Okay!

File: portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro (hunk 2 above)
Comment (gleicelisbino) [reply portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro:971]: @natocesarrego I tried to do this, but it didn't work.

**Pull request closed without merging**